### PR TITLE
Adds Meoworm

### PR DESCRIPTION
**Pull request by lordpidey**
This PR adds a new simple animal, the meoworm, which is basically just a small worm, that goes around, using the OLD adminhelp sound from around six years ago.  It's designed to be harmless, but VERY obnoxious, and since they will only come from gold slime cores, they shouldn't be too common.

:cl: lordpidey
add: Gold slime cores have now been known to spawn "meoworms", a new annelida, who's call is known to strike fear into the most experienced of spacefarers.
/:cl:

### CODE REVIEW

**Review by ShizCalev — APPROVED** [review]

**Comment by AnturK** [comment]
This is the kind of thing we have varedit for.

File: code/modules/mob/living/simple_animal/friendly/meoworm.dm [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	verb_ask = "meows inquisitively"
+	verb_exclaim = "meows intensely"
+	verb_yell = "meows intensely"
+	speak_chance = 30
+
+
+/mob/living/simple_animal/meoworm/handle_automated_speech(override)
+	if(speak_chance && (override || prob(speak_chance)))
+		icon_state = "meoworm_roar"
+		playsound(src, 'sound/creatures/loud_meow.ogg', 200)
```
Comment (Jordie0608) [inline code/modules/mob/living/simple_animal/friendly/meoworm.dm]: >200
Rip headphone users.
My system volume is on 10% and I still found this to be quite loud.

**Comment by Tlaltecuhtli** [comment]
does this have anything to do with cat people

**Comment by ExcessiveUseOfCobblestone** [comment]
codersprite caterpillar

**Comment by epochayur** [comment]
wow thanks it's terrible

File: code/modules/mob/living/simple_animal/friendly/meoworm.dm (hunk 1 above)
Comment (Cyberboss) [reply code/modules/mob/living/simple_animal/friendly/meoworm.dm]: this maps directly to byond volume so i'm not sure it even does anything

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Change the volume to 100

**Comment by optimumtact** [comment]
this is stupid

**Pull request closed without merging**